recipes/ksw/build.sh

```diff
@@ -1,11 +1,33 @@
 #!/bin/bash
 
-# These are not used ksw's makefile
-#export CFLAGS="-I$PREFIX/include"
-#export LDFLAGS="-L$PREFIX/lib"
-#export CPATH=${PREFIX}/include
+set -euo pipefail
+
+# These are used ksw's makefile
+export INCLUDE_PATH="${PREFIX}/include"
+export LIBRARY_PATH="${PREFIX}/lib"
+export LD_LIBRARY_PATH="${PREFIX}/lib"
+export CFLAGS="-I$PREFIX/include"
```

Comment:
minor nit: wouldn't you want to use the same style of escaping in all the exports in this block?

Reply:
e.g. {}